evadb/executor/create_function_executor.py

```diff
@@ -255,32 +269,142 @@ def handle_forecasting_function(self):
             frequency.split("-")[0] if "-" in frequency else frequency
         )  # shortens longer frequencies like Q-DEC
         season_length = season_dict[new_freq] if new_freq in season_dict else 1
-        model = StatsForecast(
-            [model_dict[model_name](season_length=season_length)], freq=new_freq
-        )
+
+        try_to_import_forecast()
```

Comment:
We should not require user to install both nerualforecast and statsforecast. Make them separate is better. 
`try_to_import_statsforecast` and `try_to_import_neuralforecast`.